### ISSUE TITLE
Composer version constraints are incompatible with the current release of Laravel Homestead

### ISSUE THREAD

**Issue by jackwh** [svenluijten/forge-cli#40]
I've just installed the most recent release of Laravel Homestead. The default packages installed with Composer on the current release of the Homestead box prevent the installation of `forge-cli`.

After Homestead's installation completes, the global package list is as follows (via `composer global show`):

```
vagrant@homestead:~  
▶ composer global show    
Changed current directory to /home/vagrant/.composer
doctrine/inflector               1.3.1              Common String Manipulations with regard to casing and singular/plural rules.
guzzlehttp/guzzle                6.5.2              Guzzle is a PHP HTTP client library
guzzlehttp/promises              v1.3.1             Guzzle promises library
guzzlehttp/psr7                  1.6.1              PSR-7 message implementation that also provides common utility methods
hirak/prestissimo                0.3.10             composer parallel install plugin
illuminate/contracts             v7.2.2             The Illuminate Contracts package.
illuminate/filesystem            v7.2.2             The Illuminate Filesystem package.
illuminate/support               v7.2.2             The Illuminate Support package.
laravel/envoy                    v2.0.1             Elegant SSH tasks for PHP.
laravel/installer                v3.0.1             Laravel application installer.
laravel/lumen-installer          v1.1.1             Lumen application installer.
laravel/spark-installer          dev-master 4c8dcd2 Command line installer for Laravel Spark.
nesbot/carbon                    2.31.0             An API extension for DateTime that supports 281 different languages.
psr/container                    1.0.0              Common Container Interface (PHP FIG PSR-11)
psr/http-message                 1.0.1              Common interface for HTTP messages
psr/simple-cache                 1.0.1              Common interfaces for simple caching
ralouphie/getallheaders          3.0.3              A polyfill for getallheaders.
slince/composer-registry-manager 2.3.0              Composer mirrors manager
symfony/console                  v5.0.5             Symfony Console Component
symfony/filesystem               v5.0.7             Symfony Filesystem Component
symfony/finder                   v5.0.5             Symfony Finder Component
symfony/polyfill-ctype           v1.15.0            Symfony polyfill for ctype functions
symfony/polyfill-mbstring        v1.14.0            Symfony polyfill for the Mbstring extension
symfony/polyfill-php73           v1.14.0            Symfony polyfill backporting some PHP 7.3+ features to lower PHP versions
symfony/process                  v5.0.5             Symfony Process Component
symfony/service-contracts        v2.0.1             Generic abstractions related to writing services
symfony/translation              v5.0.5             Symfony Translation Component
symfony/translation-contracts    v2.0.1             Generic abstractions related to translation
voku/portable-ascii              1.4.10             Portable ASCII library - performance optimized (ascii) string functions for php.
```

Trying to install this package with the command `composer global require "sven/forge-cli"` results in the following errors:

```
vagrant@homestead:~  
▶ composer global require "sven/forge-cli"        
Changed current directory to /home/vagrant/.composer
    1/5:        http://repo.packagist.org/p/provider-2020-04$83eb96e62de97b80ce3a7f173873660ec6f9db32c5357bc653cec40330640d62.json
    2/5:        http://repo.packagist.org/p/provider-latest$9047db88536a2f7e003a5eafcdf648f6b0c7274403182dcf31c0956a0a8be997.json
    3/5:        http://repo.packagist.org/p/provider-2017$9721b1ccb84023cd86e0d0c90ce50a3cb8f7595cf190d9c8505ca36009b1888c.json
    4/5:        http://repo.packagist.org/p/provider-2019-10$6dc0b4e5c117252c845145a27465cb969db3796f5202e616995aab830281bde9.json
    5/5:        http://repo.packagist.org/p/provider-2020-01$93d844837a76def4c4cf524c204c223b61861818f731b88a80597785bb795a9c.json
    Finished: success: 5, skipped: 0, failure: 0, total: 5
Using version ^0.4.0 for sven/forge-cli
./composer.json has been updated
Loading composer repositories with package information
Updating dependencies (including require-dev)
Your requirements could not be resolved to an installable set of packages.

  Problem 1
    - Installation request for sven/forge-cli ^0.4.0 -> satisfiable by sven/forge-cli[v0.4.0].
    - Conclusion: remove symfony/console v5.0.5
    - Conclusion: don't install symfony/console v5.0.5
    - sven/forge-cli v0.4.0 requires symfony/console ^4.0 || ^3.0 -> satisfiable by symfony/console[v3.0.0, v3.0.1, v3.0.2, v3.0.3, v3.0.4, v3.0.5, v3.0.6, v3.0.7, v3.0.8, v3.0.9, v3.1.0, v3.1.1, v3.1.10, v3.1.2, v3.1.3, v3.1.4, v3.1.5, v3.1.6, v3.1.7, v3.1.8, v3.1.9, v3.2.0, v3.2.1, v3.2.10, v3.2.11, v3.2.12, v3.2.13, v3.2.14, v3.2.2, v3.2.3, v3.2.4, v3.2.5, v3.2.6, v3.2.7, v3.2.8, v3.2.9, v3.3.0, v3.3.1, v3.3.10, v3.3.11, v3.3.12, v3.3.13, v3.3.14, v3.3.15, v3.3.16, v3.3.17, v3.3.18, v3.3.2, v3.3.3, v3.3.4, v3.3.5, v3.3.6, v3.3.7, v3.3.8, v3.3.9, v3.4.0, v3.4.1, v3.4.10, v3.4.11, v3.4.12, v3.4.13, v3.4.14, v3.4.15, v3.4.16, v3.4.17, v3.4.18, v3.4.19, v3.4.2, v3.4.20, v3.4.21, v3.4.22, v3.4.23, v3.4.24, v3.4.25, v3.4.26, v3.4.27, v3.4.28, v3.4.29, v3.4.3, v3.4.30, v3.4.31, v3.4.32, v3.4.33, v3.4.34, v3.4.35, v3.4.36, v3.4.37, v3.4.38, v3.4.39, v3.4.4, v3.4.5, v3.4.6, v3.4.7, v3.4.8, v3.4.9, v4.0.0, v4.0.1, v4.0.10, v4.0.11, v4.0.12, v4.0.13, v4.0.14, v4.0.15, v4.0.2, v4.0.3, v4.0.4, v4.0.5, v4.0.6, v4.0.7, v4.0.8, v4.0.9, v4.1.0, v4.1.1, v4.1.10, v4.1.11, v4.1.12, v4.1.2, v4.1.3, v4.1.4, v4.1.5, v4.1.6, v4.1.7, v4.1.8, v4.1.9, v4.2.0, v4.2.1, v4.2.10, v4.2.11, v4.2.12, v4.2.2, v4.2.3, v4.2.4, v4.2.5, v4.2.6, v4.2.7, v4.2.8, v4.2.9, v4.3.0, v4.3.1, v4.3.10, v4.3.11, v4.3.2, v4.3.3, v4.3.4, v4.3.5, v4.3.6, v4.3.7, v4.3.8, v4.3.9, v4.4.0, v4.4.1, v4.4.2, v4.4.3, v4.4.4, v4.4.5, v4.4.6, v4.4.7].
    - Can only install one of: symfony/console[v4.3.0, v5.0.5].
    - Can only install one of: symfony/console[v4.3.1, v5.0.5].
    - Can only install one of: symfony/console[v4.3.10, v5.0.5].
    - Can only install one of: symfony/console[v4.3.11, v5.0.5].
    - Can only install one of: symfony/console[v4.3.2, v5.0.5].
    - Can only install one of: symfony/console[v4.3.3, v5.0.5].
    - Can only install one of: symfony/console[v4.3.4, v5.0.5].
    - Can only install one of: symfony/console[v4.3.5, v5.0.5].
    - Can only install one of: symfony/console[v4.3.6, v5.0.5].
    - Can only install one of: symfony/console[v4.3.7, v5.0.5].
    - Can only install one of: symfony/console[v4.3.8, v5.0.5].
    - Can only install one of: symfony/console[v4.3.9, v5.0.5].
    - Can only install one of: symfony/console[v4.4.0, v5.0.5].
    - Can only install one of: symfony/console[v4.4.1, v5.0.5].
    - Can only install one of: symfony/console[v4.4.2, v5.0.5].
    - Can only install one of: symfony/console[v4.4.3, v5.0.5].
    - Can only install one of: symfony/console[v4.4.4, v5.0.5].
    - Can only install one of: symfony/console[v4.4.5, v5.0.5].
    - Can only install one of: symfony/console[v4.4.6, v5.0.5].
    - Can only install one of: symfony/console[v4.4.7, v5.0.5].
    - Can only install one of: symfony/console[v3.0.0, v5.0.5].
    - Can only install one of: symfony/console[v3.0.1, v5.0.5].
    - Can only install one of: symfony/console[v3.0.2, v5.0.5].
    - Can only install one of: symfony/console[v3.0.3, v5.0.5].
    - Can only install one of: symfony/console[v3.0.4, v5.0.5].
    - Can only install one of: symfony/console[v3.0.5, v5.0.5].
    - Can only install one of: symfony/console[v3.0.6, v5.0.5].
    - Can only install one of: symfony/console[v3.0.7, v5.0.5].
    - Can only install one of: symfony/console[v3.0.8, v5.0.5].
    - Can only install one of: symfony/console[v3.0.9, v5.0.5].
    - Can only install one of: symfony/console[v3.1.0, v5.0.5].
    - Can only install one of: symfony/console[v3.1.1, v5.0.5].
    - Can only install one of: symfony/console[v3.1.10, v5.0.5].
    - Can only install one of: symfony/console[v3.1.2, v5.0.5].
    - Can only install one of: symfony/console[v3.1.3, v5.0.5].
    - Can only install one of: symfony/console[v3.1.4, v5.0.5].
    - Can only install one of: symfony/console[v3.1.5, v5.0.5].
    - Can only install one of: symfony/console[v3.1.6, v5.0.5].
    - Can only install one of: symfony/console[v3.1.7, v5.0.5].
    - Can only install one of: symfony/console[v3.1.8, v5.0.5].
    - Can only install one of: symfony/console[v3.1.9, v5.0.5].
    - Can only install one of: symfony/console[v3.2.0, v5.0.5].
    - Can only install one of: symfony/console[v3.2.1, v5.0.5].
    - Can only install one of: symfony/console[v3.2.10, v5.0.5].
    - Can only install one of: symfony/console[v3.2.11, v5.0.5].
    - Can only install one of: symfony/console[v3.2.12, v5.0.5].
    - Can only install one of: symfony/console[v3.2.13, v5.0.5].
    - Can only install one of: symfony/console[v3.2.14, v5.0.5].
    - Can only install one of: symfony/console[v3.2.2, v5.0.5].
    - Can only install one of: symfony/console[v3.2.3, v5.0.5].
    - Can only install one of: symfony/console[v3.2.4, v5.0.5].
    - Can only install one of: symfony/console[v3.2.5, v5.0.5].
    - Can only install one of: symfony/console[v3.2.6, v5.0.5].
    - Can only install one of: symfony/console[v3.2.7, v5.0.5].
    - Can only install one of: symfony/console[v3.2.8, v5.0.5].
    - Can only install one of: symfony/console[v3.2.9, v5.0.5].
    - Can only install one of: symfony/console[v3.3.0, v5.0.5].
    - Can only install one of: symfony/console[v3.3.1, v5.0.5].
    - Can only install one of: symfony/console[v3.3.10, v5.0.5].
    - Can only install one of: symfony/console[v3.3.11, v5.0.5].
    - Can only install one of: symfony/console[v3.3.12, v5.0.5].
    - Can only install one of: symfony/console[v3.3.13, v5.0.5].
    - Can only install one of: symfony/console[v3.3.14, v5.0.5].
    - Can only install one of: symfony/console[v3.3.15, v5.0.5].
    - Can only install one of: symfony/console[v3.3.16, v5.0.5].
    - Can only install one of: symfony/console[v3.3.17, v5.0.5].
    - Can only install one of: symfony/console[v3.3.18, v5.0.5].
    - Can only install one of: symfony/console[v3.3.2, v5.0.5].
    - Can only install one of: symfony/console[v3.3.3, v5.0.5].
    - Can only install one of: symfony/console[v3.3.4, v5.0.5].
    - Can only install one of: symfony/console[v3.3.5, v5.0.5].
    - Can only install one of: symfony/console[v3.3.6, v5.0.5].
    - Can only install one of: symfony/console[v3.3.7, v5.0.5].
    - Can only install one of: symfony/console[v3.3.8, v5.0.5].
    - Can only install one of: symfony/console[v3.3.9, v5.0.5].
    - Can only install one of: symfony/console[v3.4.0, v5.0.5].
    - Can only install one of: symfony/console[v3.4.1, v5.0.5].
    - Can only install one of: symfony/console[v3.4.10, v5.0.5].
    - Can only install one of: symfony/console[v3.4.11, v5.0.5].
    - Can only install one of: symfony/console[v3.4.12, v5.0.5].
    - Can only install one of: symfony/console[v3.4.13, v5.0.5].
    - Can only install one of: symfony/console[v3.4.14, v5.0.5].
    - Can only install one of: symfony/console[v3.4.15, v5.0.5].
    - Can only install one of: symfony/console[v3.4.16, v5.0.5].
    - Can only install one of: symfony/console[v3.4.17, v5.0.5].
    - Can only install one of: symfony/console[v3.4.18, v5.0.5].
    - Can only install one of: symfony/console[v3.4.19, v5.0.5].
    - Can only install one of: symfony/console[v3.4.2, v5.0.5].
    - Can only install one of: symfony/console[v3.4.20, v5.0.5].
    - Can only install one of: symfony/console[v3.4.21, v5.0.5].
    - Can only install one of: symfony/console[v3.4.22, v5.0.5].
    - Can only install one of: symfony/console[v3.4.23, v5.0.5].
    - Can only install one of: symfony/console[v3.4.24, v5.0.5].
    - Can only install one of: symfony/console[v3.4.25, v5.0.5].
    - Can only install one of: symfony/console[v3.4.26, v5.0.5].
    - Can only install one of: symfony/console[v3.4.27, v5.0.5].
    - Can only install one of: symfony/console[v3.4.28, v5.0.5].
    - Can only install one of: symfony/console[v3.4.29, v5.0.5].
    - Can only install one of: symfony/console[v3.4.3, v5.0.5].
    - Can only install one of: symfony/console[v3.4.30, v5.0.5].
    - Can only install one of: symfony/console[v3.4.31, v5.0.5].
    - Can only install one of: symfony/console[v3.4.32, v5.0.5].
    - Can only install one of: symfony/console[v3.4.33, v5.0.5].
    - Can only install one of: symfony/console[v3.4.34, v5.0.5].
    - Can only install one of: symfony/console[v3.4.35, v5.0.5].
    - Can only install one of: symfony/console[v3.4.36, v5.0.5].
    - Can only install one of: symfony/console[v3.4.37, v5.0.5].
    - Can only install one of: symfony/console[v3.4.38, v5.0.5].
    - Can only install one of: symfony/console[v3.4.39, v5.0.5].
    - Can only install one of: symfony/console[v3.4.4, v5.0.5].
    - Can only install one of: symfony/console[v3.4.5, v5.0.5].
    - Can only install one of: symfony/console[v3.4.6, v5.0.5].
    - Can only install one of: symfony/console[v3.4.7, v5.0.5].
    - Can only install one of: symfony/console[v3.4.8, v5.0.5].
    - Can only install one of: symfony/console[v3.4.9, v5.0.5].
    - Can only install one of: symfony/console[v4.0.0, v5.0.5].
    - Can only install one of: symfony/console[v4.0.1, v5.0.5].
    - Can only install one of: symfony/console[v4.0.10, v5.0.5].
    - Can only install one of: symfony/console[v4.0.11, v5.0.5].
    - Can only install one of: symfony/console[v4.0.12, v5.0.5].
    - Can only install one of: symfony/console[v4.0.13, v5.0.5].
    - Can only install one of: symfony/console[v4.0.14, v5.0.5].
    - Can only install one of: symfony/console[v4.0.15, v5.0.5].
    - Can only install one of: symfony/console[v4.0.2, v5.0.5].
    - Can only install one of: symfony/console[v4.0.3, v5.0.5].
    - Can only install one of: symfony/console[v4.0.4, v5.0.5].
    - Can only install one of: symfony/console[v4.0.5, v5.0.5].
    - Can only install one of: symfony/console[v4.0.6, v5.0.5].
    - Can only install one of: symfony/console[v4.0.7, v5.0.5].
    - Can only install one of: symfony/console[v4.0.8, v5.0.5].
    - Can only install one of: symfony/console[v4.0.9, v5.0.5].
    - Can only install one of: symfony/console[v4.1.0, v5.0.5].
    - Can only install one of: symfony/console[v4.1.1, v5.0.5].
    - Can only install one of: symfony/console[v4.1.10, v5.0.5].
    - Can only install one of: symfony/console[v4.1.11, v5.0.5].
    - Can only install one of: symfony/console[v4.1.12, v5.0.5].
    - Can only install one of: symfony/console[v4.1.2, v5.0.5].
    - Can only install one of: symfony/console[v4.1.3, v5.0.5].
    - Can only install one of: symfony/console[v4.1.4, v5.0.5].
    - Can only install one of: symfony/console[v4.1.5, v5.0.5].
    - Can only install one of: symfony/console[v4.1.6, v5.0.5].
    - Can only install one of: symfony/console[v4.1.7, v5.0.5].
    - Can only install one of: symfony/console[v4.1.8, v5.0.5].
    - Can only install one of: symfony/console[v4.1.9, v5.0.5].
    - Can only install one of: symfony/console[v4.2.0, v5.0.5].
    - Can only install one of: symfony/console[v4.2.1, v5.0.5].
    - Can only install one of: symfony/console[v4.2.10, v5.0.5].
    - Can only install one of: symfony/console[v4.2.11, v5.0.5].
    - Can only install one of: symfony/console[v4.2.12, v5.0.5].
    - Can only install one of: symfony/console[v4.2.2, v5.0.5].
    - Can only install one of: symfony/console[v4.2.3, v5.0.5].
    - Can only install one of: symfony/console[v4.2.4, v5.0.5].
    - Can only install one of: symfony/console[v4.2.5, v5.0.5].
    - Can only install one of: symfony/console[v4.2.6, v5.0.5].
    - Can only install one of: symfony/console[v4.2.7, v5.0.5].
    - Can only install one of: symfony/console[v4.2.8, v5.0.5].
    - Can only install one of: symfony/console[v4.2.9, v5.0.5].
    - Installation request for symfony/console (locked at v5.0.5) -> satisfiable by symfony/console[v5.0.5].


Installation failed, reverting ./composer.json to its original content.
✘ 2
```

**Comment by jackwh** [comment]
Pull request #41 fixes this issue.

**Comment by svenluijten** [comment]
Fixed in #41. Thanks again @JackWH!
